api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java

```diff
@@ -242,14 +242,10 @@ private RdrResearcher toRdrResearcher(DbUser dbUser) {
       researcher.setLgbtqIdentity(dbDemographicSurvey.getLgbtqIdentity());
       researcher.setIdentifiesAsLgbtq(dbDemographicSurvey.getIdentifiesAsLgbtq());
     }
-    researcher.setAffiliations(
-        dbUser.getInstitutionalAffiliations().stream()
-            .map(
-                inst ->
-                    new ResearcherAffiliation()
-                        .institution(inst.getInstitution())
-                        .role(inst.getRole()))
-            .collect(Collectors.toList()));
+
+    // Deprecated old-style institutional affiliations
+    // To be removed in RW-4362
+    researcher.setAffiliations(Collections.emptyList());
```

Comment:
Confirmed with RDR that it's OK to stop sending this - it has been bad information for a long time already.